consensus/src/ordered_broadcast/engine.rs

```diff
@@ -497,7 +503,7 @@ impl<
         let partial = ops::partial_sign_message(
```

Comment:
We should consider moving signing into `types`. It feels weird to have to do this message stuff manually out here...

Reply:
We should do the same for `threshold-simplex` if we go this route.